### PR TITLE
[quant][pt2e] Add choose_qparams ops in quant namespace, to be used in representing dynamically quantized linear

### PR DESCRIPTION
**Pull request by jerryzh168**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #106992

Summary:
att

Test Plan:
will be tested later

Reviewers:

Subscribers:

Tasks:

Tags:

### CODE REVIEW

**Review by andrewor14 — APPROVED** [review]

File: torch/ao/quantization/pt2e/representation/_ops.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
+from typing import Tuple
+
+import torch
+from torch.ao.quantization.utils import determine_qparams, validate_qmin_qmax
+from torch.library import impl, Library
+
+quant_lib = Library("quant", "DEF")
```
Comment (kimishpatel) [inline torch/ao/quantization/pt2e/representation/_ops.py:7]: why are we not adding this to quantized_decomposed lib? why a separte lib?

File: torch/ao/quantization/pt2e/representation/_ops.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+
+quant_lib.define(
+    "choose_qparams_symmetric(Tensor input, int quant_min, int quant_max, "
+    "float eps, ScalarType dtype) -> (Tensor, Tensor)"
+)
+
+
+@impl(quant_lib, "choose_qparams_symmetric", "CompositeExplicitAutograd")
+def choose_qparams_symmetric(
```
Comment (kimishpatel) [inline torch/ao/quantization/pt2e/representation/_ops.py:60]: Does this need to be a separate op?

**Review by kimishpatel — CHANGES_REQUESTED** [review]
Left some comments around the need for op namespace and need for different ops for symmetric vs. asymmetric

File: torch/ao/quantization/pt2e/representation/_ops.py (hunk 1 above)
Comment (jerryzh168) [reply torch/ao/quantization/pt2e/representation/_ops.py:7]: that namespace is a bit too long, we may want to deprecate that

Comment (kimishpatel) [reply torch/ao/quantization/pt2e/representation/_ops.py:7]: Ok, so lets pick one. For the comment, ok with the change for now, but lets converge on one name.

File: torch/ao/quantization/pt2e/representation/_ops.py (hunk 2 above)
Comment (jerryzh168) [reply torch/ao/quantization/pt2e/representation/_ops.py:60]: yeah, I think we tried to use one op before (with qscheme being an argument), but looks like qscheme is not traceable or something

**Comment by jerryzh168** [comment]
discussed offline, we'll rename quantized_decomposed namespace to quant namespace (concrete name can change) in the future as a single bc-breaking change

**Pull request closed without merging**